_submissions/round-12/1/2015-02-26-shell_scripts.md

```diff
@@ -0,0 +1,15 @@
+--
+date: 2015-02-26
+round: Round 12
+title: Concept Maps for Shell Scripts
+author: Dorota Jarecka
+permalink: /2015/02/shell_scripts-concept_maps/
+tags:
+  - shell
+  - script
+  - concept map
+---
+
+Concept map for shell scripting based on [the Software Carpentry lessons](http://swcarpentry.github.io/shell-novice/05-script.html).
+
+Please see the [picture of my concept map](http://i.imgur.com/KURmyog.jpg).
```

Comment:
Markdown uses `![title](link)` (with a `!` in front) to embed an image.  I'll add the `!` after I merge.